### DISCUSSION TITLE
Unterschiedliche ESP Files

### DISCUSSION THREAD

**Discussion by Yosh1977** [tbnobody/OpenDTU#1143] [General]
Hallo,

kann mir einer sagen warum es nun die Unterschiedlichen ESP Files gibt ?
Auf meinem ESP32 mit 38 Pins kann ich das normale bin file laden oder das ESP32 File. Beides funktioniert.
Was sind da die Unterschiede.

Besten Dank

**Comment by Solarteur** [comment]
Ist hier
https://github.com/tbnobody/OpenDTU/discussions/1090
beschrieben.
